src/app/services/technical-record/technical-record.service.ts

```diff
@@ -64,36 +66,32 @@ export class TechnicalRecordService {
 
   putUpdateTechRecords(
     systemNumber: string,
-    techRecord: TechRecordModel,
+    vehicleTechRecord: VehicleTechRecordModel,
     user: { id?: string; name: string },
     recordToArchiveStatus?: StatusCodes,
     newStatus?: StatusCodes
   ) {
-    const newTechRecord = cloneDeep(techRecord);
+    const newVehicleTechRecord = cloneDeep(vehicleTechRecord);
+    const newTechRecord = newVehicleTechRecord.techRecord[0];
     newTechRecord.statusCode = newStatus ?? newTechRecord.statusCode;
-    this.removeUpdateType(techRecord, newTechRecord);
+    delete newTechRecord.updateType;
 
     const url = `${environment.VTM_API_URI}/vehicles/${systemNumber}` + `${recordToArchiveStatus ? '?oldStatusCode=' + recordToArchiveStatus : ''}`;
 
-    const body = {
+    const body: PutVehicleTechRecordModel & { msUserDetails: { msOid: string | undefined; msUser: string } } = {
+      ...this.formatVrmsForUpdatePayload(vehicleTechRecord),
       msUserDetails: { msOid: user.id, msUser: user.name },
       techRecord: [newTechRecord]
     };
 
     return this.http.put<VehicleTechRecordModel>(url, body, { responseType: 'json' });
   }
 
-  private removeUpdateType(techRecord: TechRecordModel, newTechRecord: TechRecordModel) {
```

Comment:
Looked to me like this method was redundant, both functions that use it pass `newTechRecord` as a deep clone of `techRecord`, with minor modifications to the `newTechRecord` object before it gets passed in, however it doesn't seem any of the changed properties have an impact on the value of `newTechRecord.updateType` or `techRecord.updateType`.